bin/clover/deno.json

```diff
@@ -4,6 +4,21 @@
     "updateSchema": "../../lib/jsr-systeminit/cf-db/update-schema.sh",
     "run": "deno run --allow-all main.ts"
   },
+  "fmt": {
```

Comment:
Wow! I do wonder if there's a way to create a shared file for this, but I don't want to wait to figure it out, this is awesome :)